openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -63,12 +111,13 @@ def _post_body(self, user, client, token_type=None):
 
 
 @ddt.ddt
-class TestAccessTokenView(mixins.AccessTokenMixin, _DispatchingViewTestCase):
+class TestAccessTokenView(mixins.AccessTokenMixin, _DispatchingViewTestCase, AccessTokenLoginMixin):
     """
     Test class for AccessTokenView
     """
     def setUp(self):
         self.url = reverse('access_token')
+        self.login_with_access_token_url = reverse("login_with_access_token")
```

Comment:
This line shouldn't be needed here since it's set in the `setUp` method of `AccessTokenLoginMixin`.